src/commcare_cloud/ansible/group_vars/all.yml

```diff
@@ -183,6 +183,7 @@ ECRYPTFS_PASSWORD: "{{ secrets.ECRYPTFS_PASSWORD | default(None) }}"
 KSPLICE_ACTIVATION_KEY: "{{ secrets.KSPLICE_ACTIVATION_KEY | default(None) }}"
 NEW_RELIC_KEY: "{{ secrets.NEW_RELIC_KEY | default(None) }}"
 
+# use https://v-punk.com/automate-password-changes-with-ansible/ to generate hash
```

Comment:
It's actually much easier now that ansible has a `password_hash` filter: https://stackoverflow.com/questions/19292899/creating-a-new-user-and-password-with-ansible#29910294